test/runtime_test.cc

```diff
@@ -173,26 +167,5 @@ TEST_P(TestVM, Trap) {
   ASSERT_TRUE(integration_->error_message_.find(exp_message) != std::string::npos);
 }
 
-TEST_P(TestVM, WithPrecompiledSection) {
```

Comment:
is there a test that covers this case in Envoy?

Reply:
We have tests with and without precompiled section in Envoy when using V8, yes.

But I removed this primarly because it's not representative anymore, i.e. we never load "mixed" module into WasmVM anymore, we pass stripped and precompiled modules separately.